### PR TITLE
91.5% code coverage is sdc.go

### PR DESCRIPTION
**Pull request by daniyaliqbal2024**
# Description
Increase sdc.go unit test coverage past threshold.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| https://github.com/dell/csm/issues/1559 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration
image

### CODE REVIEW

File: sdc_test.go [hunk 1]
```diff
@@ -469,3 +523,639 @@ func TestSetApprovedIps(t *testing.T) {
...
+		defer tc.server.Close()
+
+		system := System{
+			System: &types.System{
+				ID: systemID,
+			},
+			client: client,
+		}
+
+		_, err = system.FindSdc("ID", searchSdcID)
```
Comment (tdawe) [inline sdc_test.go:658]: Get the response and validate it equals what we expect

File: sdc_test.go [hunk 2]
```diff
@@ -469,3 +523,639 @@ func TestSetApprovedIps(t *testing.T) {
...
+		defer tc.server.Close()
+
+		system := System{
+			System: &types.System{
+				ID: systemID,
+			},
+			client: client,
+		}
+
+		_, err = system.GetSdcByID(searchSdcID)
```
Comment (tdawe) [inline sdc_test.go:719]: Get the response and validate it equals what we expect

File: sdc_test.go [hunk 3]
```diff
@@ -469,3 +523,639 @@ func TestSetApprovedIps(t *testing.T) {
...
+		defer tc.server.Close()
+
+		system := System{
+			System: &types.System{
+				ID: systemID,
+			},
+			client: client,
+		}
+
+		_, err = system.ChangeSdcName(searchSdcID, "NameSDC")
```
Comment (tdawe) [inline sdc_test.go:780]: Get the response and validate it equals what we expect

File: sdc_test.go [hunk 4]
```diff
@@ -469,3 +523,639 @@ func TestSetApprovedIps(t *testing.T) {
...
+		defer tc.server.Close()
+
+		system := System{
+			System: &types.System{
+				ID: systemID,
+			},
+			client: client,
+		}
+
+		_, err = system.ChangeSdcPerfProfile(sdcID, "Perf Profile")
```
Comment (tdawe) [inline sdc_test.go:841]: Get the response and validate it equals what we expect

File: sdc_test.go [hunk 5]
```diff
@@ -469,3 +523,639 @@ func TestSetApprovedIps(t *testing.T) {
...
+			t.Fatal(err)
+		}
+		defer tc.server.Close()
+
+		sdc := Sdc{
+			Sdc:    &tc.sdc,
+			client: client,
+		}
+
+		_, err = sdc.GetStatistics()
```
Comment (tdawe) [inline sdc_test.go:927]: Get the response and validate it equals what we expect

File: sdc_test.go [hunk 6]
```diff
@@ -469,3 +524,679 @@ func TestSetApprovedIps(t *testing.T) {
...
+			if res.Sdc.PerfProfile != perfProfile {
+				t.Fatal(err)
+			}
+		}
+	}
+}
+
+func TestGetSatistics(t *testing.T) {
+	sdcID := uuid.NewString()
+	mapVolumes := rand.Int()
```
Comment (tdawe) [inline sdc_test.go:887]: Linting issue

**Review by gallacher — APPROVED** [review]
Please ensure all commits are signed

File: README.md [hunk 7]
```diff
@@ -7,6 +7,7 @@ The *Goscaleio* project represents API bindings that can be used to provide Scal
 - [Licensing](#licensing)
 - [Support](#support)
 
+-TEST
```
Comment (gallacher) [inline README.md:10]: Why is this here?
